### PR TITLE
added more options to scto_write.

### PR DESCRIPTION
**Pull request by jakejh**
See what you think!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# rsurveycto 0.0.6
+* Added more options to `scto_write()`.
```
Comment (roboton) [inline NEWS.md]: Add more specifics in the NEWS update?

File: R/utils.R [hunk 2]
```diff
@@ -10,6 +10,10 @@ get_csrf_token = function(servername, username, password) {
   index_res = GET(index_url)
   csrf_token = httr::headers(index_res)$`x-csrf-token`
 
+  if (is.null(csrf_token)) {
```
Comment (roboton) [inline R/utils.R:13]: Is this the correct error condition we're looking for? Did I add this? We actually want to do a check on user_type to ensure that it's paid_user vs anonymous user.

**Review by roboton — APPROVED** [review]
LGTM Jake, thanks for doing this. Tests and everything!

**Comment by roboton** [comment]
I'll leave it to you to merge @jakejh

File: R/utils.R (hunk 2 above)
Comment (jakejh) [reply R/utils.R:13]: I added it. In my testing on the rsurveycto server, if the server doesn't exist or isn't running, the csrf token will be NULL. If the server is running and I give it my correct username and password, the user_type is community_user. I'll dig into it more.

Comment (jakejh) [reply R/utils.R:13]: I'm going to merge as is and save any changes for a separate pull request.

**Comment by roboton** [comment]
oh i've never seen community_user before (maybe that's for a free scto
account?)

i have user_category=paid_user when logged in with my ***@***.*** to
young1ovedata server
and when i'm not logged in user_category=anonymous_form_filler

thanks Jake!

**Pull request merged**